Automattic/jetpack
# Add changelog 6.4

#### Changes proposed in this Pull Request:

* Updates changelog.txt to include changelog for 6.3.3
* Updates readme.txt to include changelog for 6.4
  * Also brings back pure unix line endings to `readme.txt`.
* Bumps stable tag to `6.3.3`
* Updates `to-test.md` with testing instructions from PRs/changesets shipped with this release

#### Testing instructions

* Check the [rendered readme.txt](https://github.com/Automattic/jetpack/blob/add/changelog-64/readme.txt).
* Check the [rendered changelog.txt](https://github.com/Automattic/jetpack/blob/add/changelog-64/changelog.txt).
* Check the [rendered to-test.md](https://github.com/Automattic/jetpack/blob/add/changelog-64/to-test.md).
